packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js

```diff
@@ -22,15 +22,33 @@ describe('http-core', () => {
       })));
 
     describe('logOptions', () => {
-      it('calls logger plugin', () => {
+      it('calls logger plugin if enabled', () => {
+        // flag used to restore state of verbose network logging env variable
+        const flagged = !process.env.ENABLE_VERBOSE_NETWORK_LOGGING;
+
+        // reused toggle that toggles verbose network logging env variable
+        const toggleVNL = () => {
+          if (flagged) {
+            process.env.ENABLE_VERBOSE_NETWORK_LOGGING =
+              !process.env.ENABLE_VERBOSE_NETWORK_LOGGING;
+          }
+        };
+
+        // toggle to enabled if disabled
+        toggleVNL();
+
         const spy = sinon.spy(webex.logger, 'info');
 
         return webex.request({
           service: 'hydra',
           resource: 'people/me'
         }).then(() => {
+          // toggle to disabled if originally disabled
+          toggleVNL();
           assert.called(spy);
         }).catch(() => {
```

Comment:
We shouldn't need both a `.catch` and `.then` here. The test should follow the expected path of one or the other.